docs/topics/appendix-s2i-build-process.adoc

```diff
@@ -1,16 +1,16 @@
 [[s2i-build-process]]
 = The Source-to-Image (S2I) Build Process
-link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
+link:https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
```

Comment:
As this doc is an appendix, we should mention that 4 build approaches are proposed like 6 different input type - https://www.dropbox.com/s/2j50sv2yvpa1b2x/Screenshot%202017-11-27%2017.08.27.png?dl=0

https://docs.openshift.com/container-platform/3.6/dev_guide/builds/index.html